k4MarlinWrapper/CMakeLists.txt

```diff
@@ -54,6 +54,13 @@ target_include_directories(MarlinWrapper PUBLIC
   ${LCIO_INCLUDE_DIRS}
 )
 
+# k4MarlinWrapperUtils
+gaudi_add_library(k4MarlinWrapperUtils SHARED
```

Comment:
What is the reason to split this into a separate library, and not be part of the main library?

Reply:
Ok, I see. Thanks!